xml/obs_scm_ci_workflow_integration.xml

```diff
@@ -778,6 +778,25 @@ workflow:
       </figure>
     </sect2>
 
+    <sect2 xml:id="sec.obs.obs_scm_ci_workflow_integration.setup.workflow_runs">
+      <title>Workflow Runs</title>
+
+      <para>For every SCM event, OBS record the request, and the response sent back to the SCM.
+      These records are stored in OBS in the form of Workflow runs. OBS will show details of every
+      event received from SCM. If an error occures in any of the workflow steps, the
+      workflow run will record the same response as it was sent to the SCM.</para>
```

Comment:
Sounds like 2 times the same sentence. Maybe apart from *what* you should explain *why* we are doing this and *how* this is useful to the people reading this.

Reply:
I've pushed some updates. Please let me know if you have more suggestions. :)